draft-pauly-httpbis-geoip-hint.md

```diff
@@ -73,63 +107,91 @@ For example, the header for an entry "192.0.2.5,US,US-AL,Alabaster" would be:
-   added, if appropriate. This allows servers to automatically discover when new
-   feeds and services are brought up..
+The client MAY append the client hint header in requests to the
+server after the server has explicitly opted in to receiving the
+hint, or if the client knows of specific server configurations,
+such as proxy settings, that support including the hint.
 
-If the server is acting as a forward proxy, such as a CONNECT proxy, it can use the hint
-to determine an appropriate geo-mapped IP address to use for outbound connections, or a
-client subnet to present in the EDNS0 Client Subnet extension for DNS queries {{?RFC6891}}
-{{?RFC7871}}.
-
-# Security Considerations {#security}
-
-The use of the IP Geolocation Client Hint MUST use the Sec- header prefix as recommended
-in {{!RFC8942}}.
-
-Servers MUST NOT use IP Geolocation Client Hints for making security or access-control decisions,
-as the value can be spoofed by a client. The hint is intended only for use in optimizing behavior.
+# Server Behavior
 
-The value contained in this hint SHOULD be based only on a IP Geolocation feed value for
-an IP address the client is already presenting to a server. In order to avoid disclosing
-any private information, this value MUST not be based on geolocation of the client determined
-by other means, such as physical latitude and longitude coordinates.
+Upon receiving a Geolocation Client Hint, a server can use the
+information to influence its behavior in various ways, such as
+determining the content of HTTP responses.
+
+Servers can choose to use the hint value in one of several ways,
+including:
+
+- Using the client hint information instead of consulting IP-based
+geolocation feeds.
+- Serving content that corresponds to the client’s indicated location,
+including delivering region-specific news, weather forecasts, and
+relevant advertisements.
+
+The server MUST be able to handle situations where geolocation is
+not provided in a request. Since not all web clients will send a 
+Geolocation Client Hint, the server MAY defer to alternative methods 
+such as IP-based geolocation feeds to provide said value.
+
+# Security Considerations {#sec-considerations}
+
+The use of the Geolocation Client Hint MUST use the `Sec-` header
+prefix as recommended in {{!RFC8942}}.
+
+Servers MUST NOT use Geolocation Client Hints for making security or
+access-control decisions, as the value is provided by the client with
+no additional authentication.
+
+# Privacy Considerations {#privacy}
+
+Any default value provided in this hint MUST NOT be more specific than
+the information that could be obtained from the client's IP address
+and a well-maintained map of IP ranges to locations. In particular,
+when a privacy technology such as a VPN is in use, the default value
+MUST NOT reveal information about the user's location that would
+otherwise be hidden.
+
+To prevent disclosing private information, this value MUST NOT be
```

Comment:
Similar to above, let's use prose here to avoid repeating the normative text form the Client Behavior section

```suggestion
To prevent disclosing private information, this value cannot be
```